nservicebus/dependency-injection/property-injection.md

```diff
@@ -1,23 +0,0 @@
----
-title: Property injection
-reviewed: 2020-01-24
-component: Core
-tags:
- - Dependency Injection
-redirects:
- - nservicebus/property-injection-in-handlers
```

Comment:
Should the redirect for `nservicebus/dependency-injection/property-injection` be set up if the page is removed to avoid the broken cached link?

Reply:
good point, we can redirect to the new package's documentation I'd say?